.circleci/soltest.sh

```diff
@@ -0,0 +1,94 @@
+#! /bin/bash
+#------------------------------------------------------------------------------
+# Bash script to execute the Solidity tests by CircleCI.
+#
+# The documentation for solidity is hosted at:
+#
+#     https://solidity.readthedocs.org
+#
+# ------------------------------------------------------------------------------
+# Configuration Environment Variables:
+#
+#     EVM=version_string      Specifies EVM version to compile for (such as homestead, etc)
+#     OPTIMIZE=1              Enables backend optimizer
+#     ABI_ENCODER_V2=1        Enables ABI encoder version 2
+#
+# ------------------------------------------------------------------------------
+# This file is part of solidity.
+#
+# solidity is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# solidity is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with solidity.  If not, see <http://www.gnu.org/licenses/>
+#
+# (c) 2016 solidity contributors.
+#------------------------------------------------------------------------------
+set -ex
```

Comment:
```suggestion
set -e
```

Reply:
Actually, I did that `x` one on purpose, because it helps when investigating. :)